### ISSUE TITLE
about train dataset

### ISSUE THREAD

**Issue by luoww1992** [hustvl/ViTMatte#24]
in datasetLodaer.py:
train_dataset = DataGenerator(
    data = ImageFileTrain(
        alpha_dir='path/to/alpha',
        fg_dir='path/to/fg',
        bg_dir='path/to/bg',
        root='path/to/Adobe_Image_Matting'
    ),
    phase = 'train'
)

i know the alpha_dir is mask path,  
Q1: but the fg_dir is RGB image only fg object  ? Or  is RGBA image the all image with fg and bg ?
Q2：the bg_dir is RGB image only bg without fg object mask? Or  likes bg in  AIM with fg mask?

------ can you share me a demo dataset fold with some pictures ?  thank you !

**Comment by JingfengYao** [comment]
Images in fg_dir and bg_dir are both RGB images. As for demos, maybe you can see [here](https://paperswithcode.com/dataset/composition-1k).

**Comment by luoww1992** [comment]
@JingfengYao 
Q1, so train datase likes this:
![see](https://github.com/hustvl/ViTMatte/assets/45957595/ff6b6dd6-ae1a-46e1-ad5d-44dbf6db2408)
the fg_image's background must be bg_image ?
Q2, And what i mean about demo dataset is :
 can you share me three images in fg_dir,bg_dir andalpha_dir in your dataset ?

**Comment by JingfengYao** [comment]
@luoww1992 @yuqilol A1: No. In our setting, there is no requirement for the front background to be associated.

**Comment by JingfengYao** [comment]
@luoww1992 @yuqilol 
**A1:** No, In our setting, there is no requirement for the foreground and background to be associated.  
**A2:** I can't share the demos. Because the dataset is semi-public (owned by [DIM](https://github.com/foamliu/Deep-Image-Matting?tab=readme-ov-file#dataset)). I have no right to share them personally. You should get the dataset following their instructions.
BTW, for a simple way to get familiar with matting datasets, you may check [GFM](https://github.com/JizhiziLi/GFM?tab=readme-ov-file#am-2k). (they are similar in format to DIM data, but we didn't use them in ViTMatte training)
